src/test/components/__snapshots__/ProfileCallTreeView.test.js.snap

```diff
@@ -427,7 +427,7 @@ exports[`ProfileCallTreeView with JS Allocations matches the snapshot for JS all
             >
               <div
                 aria-expanded="true"
-                aria-label="A, running time is 15ms (100%), self time is 0ms"
+                aria-label="A, running time is 15 bytes (100%), self time is 0ms"
```

Comment:
Did you expect this change?
Also I think the full string should be changed when we're dealing with allocations...
See also the self time that still has `ms` when the value is 0.

Reply:
I added a new commit with a new method for calculating the aria labels. I didn't see those in there, thanks for catching them.